hack/production.yaml

```diff
@@ -10,7 +10,14 @@ pdb:
 topologySpreadConstraints:
   - maxSkew: 1
     topologyKey: topology.kubernetes.io/zone
-    whenUnsatisfiable: ScheduleAnyway
+    whenUnsatisfiable: DoNotSchedule
```

Comment:
I guess this fails in the 1-node test-cluster in the e2e tests now.

Reply:
It should not. TopologySpread only assumes an even distribution. Evenly distributing to only one available node should work just fine ;-). But lets see.

Reply:
Looks like you are right. Did not expect that to happen. Looks like minikube and kind are is missing the default zone annotation. I will revert this as it makes no sense here anyway.